agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -317,19 +317,21 @@ main_thread::main_thread(config_file::sConfigSlave config_,
     auto now = std::chrono::steady_clock::now();
```

Comment:
Commit message:

>agent: platform_manager: use hostap interface from platform DB
>
>Get the hostap_iface from platform DB instead of the configuration file.
>
>Signed-off-by: Adam Dov <adamx.dov@intel.com>

Reply:
changed the commit message.